website/src/pages/newsletter.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Layout from '@theme/Layout';
+import styles from '@site/src/css/newsletter.module.css';
+
+const MyPage = () => {
+  const {siteConfig} = useDocusaurusContext();
+  return (
+    <Layout
+      permalink="/"
+      title={siteConfig.title}
+      description={siteConfig.tagline}
+    >
+      <div className="hero">
+        <div className="container">
+          <div className="row" >
+            <div className="col col--6">
+              <img
+                alt='Boîte aux lettres avec du courrier dedans'
+                src={useBaseUrl('img/newsletter/inbox.png')}
+                className= { styles.newsletterImage }
+              />
+            </div>
+            <div className="col col--6">
+              <h1 className="hero__title">La newsletter de Women On Rails</h1>
+              <p className="hero__subtitle">Reste à jour de l'actualité du secteur avec les infos essentielles décodées</p>
+              <div>
```

Comment:
```suggestion
              <div class="container--centered">
```
Est-ce que le formulaire ne serait pas mieux centré ? (sur mobile c'est mieux mais en desktop je ne sais pas …)

Reply:
Et en centrant tout ?
<img width="1066" alt="Capture d’écran 2020-11-10 à 20 14 14" src="https://user-images.githubusercontent.com/42354334/98722089-4e8c9c00-2391-11eb-9650-b91e42e06cec.png">

Reply:
ah ben oui c'est pas mal comme ça :+1: